File: Entity/Usuario.cs
using System;
using System.ComponentModel.DataAnnotations;

namespace Entity
{
    public class Usuario
    {
        [Key]
        public string NumeroCedula { get; set; }
        public int TipoUsuario { get; set; }
        public string TipoUsuarioNombre { get; set; } 
        public string PrimerNombre { get; set; }
        public string SegundoNombre { get; set; }
        public string PrimerApellido { get; set; }
        public string SegundoApellido { get; set; }
        public string NombreCompleto { get; set; }      
        public string UsuarioI { get; set; }
        public string Contrasena { get; set; }
        public string Telefono { get; set; }
        public string Email { get; set; }

        public void SetTipoUsuarioNombre(){
            string[] tipo = {};
            tipo[1] = "Administrador";
            tipo[2] = "Técnico";
            this.TipoUsuarioNombre = tipo[this.TipoUsuario];
        }

        public void SetNombreCompleto(){
            this.NombreCompleto = this.PrimerNombre +' '+ this.SegundoNombre +' '+ this.PrimerApellido +' '+ this.SegundoApellido;
        }
    }
}

File: webSyscsc/ClientApp/src/app/syscsc/models/usuario.ts
export class Usuario {
    numeroCedula: string;
    tipoUsuario: number;
    tipoUsuarioNombre: string;
    primerNombre: string;
    segundoNombre: string;
    primerApellido: string;
    segundoApellido: string;
    nombreCompleto: string;
    usuarioI: string;
    contrasena: string;
    confirmarContrasena: string;
    telefono: string;
    email: string;   

    setTipoUsuarioNombre(){
        let tipo = [];
        tipo[1] = 'Administrador';
        tipo[2] = 'Técnico';
        return tipo[this.tipoUsuario];
    }

    setNombreCompleto(){
        return this.primerNombre +' '+ this.segundoNombre +' '+ this.primerApellido +' '+ this.segundoApellido;
    }

    validarContrasena(){
        if(this.contrasena == this.confirmarContrasena){
            return true;
        }
        return false;
    }
}